src/common/object.ts

```diff
@@ -36,3 +37,11 @@ export const validateRegex = (
 export const nameRegex = new RegExp(/^(.){1,60}$/);
 export const handleRegex = new RegExp(/^@?([\w-]){1,39}$/i);
 export const descriptionRegex = new RegExp(/^(.){1,250}$/);
+
+export const validateRegexOrFail = (params: ValidateRegex[]): void => {
+  const regexResult = validateRegex(params);
+
+  if (Object.keys(regexResult).length) {
+    throw new ValidationError(JSON.stringify(regexResult));
+  }
+};
```

Comment:
We already have a function for this.
I'd suggest we either use that or revamp the other to this?

You can find it as `validateRegex`

Reply:
Yes, this is it. It is using `validateRegex`. This just abstracted the throwing of error since we automatically do that.

Reply:
This is like the typeorm's function `findOne` and `findOneOrFail`.

Reply:
Yes I understand, but this is not exactly doing what those 2 other implementations do, so maybe we should convert those into this function 🙏

Reply:
Cool. Let me update this existing function to automatically throw since we do that anyways.